$DragonFly: src/secure/usr.sbin/sshd/Attic/sshd_config.no_obj.patch,v 1.5 2006/02/13 14:47:58 corecode Exp $
--- ../../../crypto/openssh-4/sshd_config	2005-09-08 01:40:57.000000000 +0200
+++ sshd_config	2005-09-08 14:51:08.000000000 +0200
@@ -1,4 +1,5 @@
 #	$OpenBSD: sshd_config,v 1.72 2005/07/25 11:59:40 markus Exp $
+#	$DragonFly: src/secure/usr.sbin/sshd/Attic/sshd_config.no_obj.patch,v 1.5 2006/02/13 14:47:58 corecode Exp $
 
 # This is the sshd server system-wide configuration file.  See
 # sshd_config(5) for more information.
@@ -10,8 +11,13 @@
 # possible, but leave them commented.  Uncommented options change a
 # default value.
 
+# Note that some of DragonFly's defaults differ from OpenBSD's, and
+# DragonFly has a few additional options.
+
+#VersionAddendum DragonFly-20060212
+
 #Port 22
-#Protocol 2,1
+#Protocol 2
 #AddressFamily any
 #ListenAddress 0.0.0.0
 #ListenAddress ::
@@ -19,7 +25,6 @@
 # HostKey for protocol version 1
 #HostKey /etc/ssh/ssh_host_key
 # HostKeys for protocol version 2
-#HostKey /etc/ssh/ssh_host_rsa_key
 #HostKey /etc/ssh/ssh_host_dsa_key
 
 # Lifetime and size of ephemeral version 1 server key
@@ -34,7 +39,7 @@
 # Authentication:
 
 #LoginGraceTime 2m
-#PermitRootLogin yes
+#PermitRootLogin no
 #StrictModes yes
 #MaxAuthTries 6
 
@@ -56,7 +61,7 @@
 #PasswordAuthentication yes
 #PermitEmptyPasswords no
 
-# Change to no to disable s/key passwords
+# Change to no to disable OPIE and PAM authentication
 #ChallengeResponseAuthentication yes
 
 # Kerberos options
@@ -81,7 +86,7 @@
 
 #AllowTcpForwarding yes
 #GatewayPorts no
-#X11Forwarding no
+#X11Forwarding yes
 #X11DisplayOffset 10
 #X11UseLocalhost yes
 #PrintMotd yes
@@ -97,6 +102,8 @@
 #PidFile /var/run/sshd.pid
 #MaxStartups 10
 
+#XAuthLocation /usr/pkg/xorg/bin/xauth
+
 # no default banner path
 #Banner /some/path
 
